### DISCUSSION TITLE
How to Prepend Cache Statement Before Select Clause?

### DISCUSSION THREAD

**Discussion by azazlhr** [maxtoroq/DbExtensions#76] [Q&A] [answered]
Hi, 
I'm using this great library to build SQL statements to avoid SQL injections; so far so good.

But now I want to create a SQL statement to support caching in SQLite DB.

**What I need:**
"Cache [CacheTableName] select * from Customers".

**What I get:**
"Select Cache [CacheTableName] * from Customers"

**Code snippet**
`
var queryBuilder = new SqlBuilder()
	.SELECT("Cache [CacheTableName] *")
	.FROM("Customers");
`

I understand, that's not a right way but how can I prepend a Cache Clause before select statement?

Thanks in advance for any solution or workaround.

**Comment by maxtoroq** [comment, marked as answer]
Use AppendClause:

```csharp
var queryBuilder = new SqlBuilder()
   .AppendClause("CACHE", null, "[CacheTableName]")
   .SELECT("*")
   ;
```
